### ISSUE TITLE
spec validation fails after OS X 10.9 update

### ISSUE THREAD

**Issue by BlackBears** [CocoaPods/CocoaPods#1507]
## Issue

Multiple specs that previously linted without errors no longer pass (or even complete). 
## Context

Mac OS X 10.9
Xcode 5.0.1 (5A2053)
Cocoapods version -> 0.26.2
which ruby ->`/usr/local/rvm/rubies/ruby-1.9.3-p374/bin/ruby`
ls -l /tmp -> `lrwxr-xr-x@ 1 root  wheel  11 Oct 23 03:25 /tmp -> private/tmp`

command line tools installed
## Example

For example, our private spec `AKDFoundation` no longer passes.  Running `pod spec lint AKDFoundation.podspec --verbose` shows that everything completes normally up to "Building with xcodebuild", at which point the process hangs, sometimes after launching numerous (>= 10) clang instances.

Running the same command as sudo, allows the build to proceed but with hundreds of errors, e.g.

```
** BUILD FAILED **


The following build commands failed:
ProcessPCH /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/com.apple.DeveloperTools/5.0.1-5A2053/Xcode/SharedPrecompiledHeaders/Pods-AKDFoundation-prefix-fomsgjyghijyqpgvogyvpqixzrtu/Pods-AKDFoundation-prefix.pch.pch Pods-AKDFoundation-prefix.pch normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
ProcessPCH /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/C/com.apple.DeveloperTools/5.0.1-5A2053/Xcode/SharedPrecompiledHeaders/Pods-AKDFoundation-prefix-dmyotstcwykltpaqqxxzwylagtbb/Pods-AKDFoundation-prefix.pch.pch Pods-AKDFoundation-prefix.pch normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler
 (2 failures)
 -> AKDFoundation (0.0.2)
- ERROR | [xcodebuild]  /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS7.0.sdk/usr/include/dispatch/dispatch.h:25:10: fatal error: cyclic dependency in module 'Darwin': Darwin -> Dispatch -> Darwin
- ERROR | [xcodebuild]  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk/System/Library/Frameworks/Security.framework/Headers/cssmconfig.h:29:10: fatal error: could not build module 'Darwin'
- ERROR | [xcodebuild]  /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk/System/Library/Frameworks/CoreServices.framework/Headers/CoreServices.h:19:10: fatal error: could not build module 'CoreFoundation
```

In short, the errors are all related to building OS frameworks, 'Darwin', 'Foundation', 'UIKit', etc. etc.  No such errors are encountered when building projects from Xcode or with xcodebuild from the command line.
## Attempts to resolve

Reinstall Cocoapods
Repaired permissions on /tmp
Implemented [this workaround](https://gist.github.com/goshakkk/5763489)
## Resolution (partial)

was able to get it to pass validation only by adding: `spec.xcconfig = { 'CLANG_ENABLE_MODULES' => 'NO' }` to the specification.
